docs/en/connector-v2/source/Jdbc.md

```diff
@@ -60,6 +60,7 @@ supports query SQL and can achieve projection effect.
 | table_list                                 | Array   | No       | -               | The list of tables to be read, you can use this configuration instead of `table_path`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
 | where_condition                            | String  | No       | -               | Common row filter conditions for all tables/queries, must start with `where`. for example `where id > 100`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | split.size                                 | Int     | No       | 8096            | How many rows in one split, captured tables are split into multiple splits when read of table.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| split.hash-function                        | String  | No       | -               | Which hash function use to split. <br/> Default is `MD5`, Oracle is `ORA_HASH`, PostgreSQL  is `HASHTEXT`.<br/>  If the default function is not work, you can use this parameter to customize the hash function.                                                                                                                                                                                                                                                                                                                                                                                                                                   |   |
```

Comment:
Looking back, is it necessary to add this parameter?

Reply:
i will remove this parameter